config.toml

```diff
@@ -23,10 +23,14 @@ disableKinds = [ "RSS", "sitemap", "taxonomy", "taxonomyTerm" ]
     name = "Wiki"
     url = "https://wiki.nordix.org/"
     weight = 5
+[[menu.main]]
+    name = "Nordix Open Labs"
```

Comment:
I think this should be nolabs as the "Nordix Open Labs" will be too long on page

Reply:
Yeah, I wasn't sure what to put there.

nolabs might not make sense to people and Nordix Open Labs is too long so I opted for the long name to be explicit.

Reply:
Thats OK with me